lib/src/core/provider/provider.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:firebase_auth/firebase_auth.dart';
```

Comment:
rename file name from provider.dart to user_provider.dart

to avoid misundferstanding in future